ui/src/components/ui/toaster.tsx

```diff
@@ -7,36 +7,32 @@
  * SPDX-License-Identifier: MIT
  */
 
-import {
-  Toast,
-  ToastClose,
-  ToastDescription,
-  ToastProvider,
-  ToastTitle,
-  ToastViewport,
-} from '@/components/ui/toast';
-import { useToast } from '@/components/ui/use-toast';
+import { useTheme } from 'next-themes';
+import { Toaster as Sonner } from 'sonner';
 
-export function Toaster() {
-  const { toasts } = useToast();
+type ToasterProps = React.ComponentProps<typeof Sonner>;
+
+const Toaster = ({ ...props }: ToasterProps) => {
+  const { theme = 'system' } = useTheme();
```

Comment:
I'd remove this for as long as themes are not supported.

Reply:
There are also [Vite specific theming instructions](https://ui.shadcn.com/docs/dark-mode/vite) on shadcn-ui, which don't imply importing any external theme libraries.

Reply:
:+1: for dark theme 😁